src/voloctree/voloctree.cpp

```diff
@@ -1955,7 +1955,7 @@ void VolOctree::_updateAdjacencies()
 
 	// Face information
 	int nCellFaces = m_cellTypeInfo->nFaces;
-	uint8_t oppositeFace[nCellFaces];
+	uint8_t oppositeFace[6];
```

Comment:
Can we do this ```const uint8_t *oppositeFace = m_tree->getOppface()``` instead? In this way we don't even need to copy the array.

Reply:
Sure, on it.